doc/base/classes.xml

```diff
@@ -22146,7 +22150,7 @@
 			<argument index="0" name="id" type="int">
 			</argument>
 			<description>
-				Return the status of hostname queued for resolving, given its queue ID. Returned status can be any of the RESOLVER_STATUS_* enumeration.
+				Returns the status of the hostname queued for resolving, given its queue ID. Returned status can be any of the RESOLVER_STATUS_* enumerators.
```

Comment:
`Returned status can be any of the RESOLVER_STATUS_* enumerators.` -> `Returns one of the RESOLVER_STATUS_* constants.`
`enumerators -> enum`